pygmo/docstrings.cpp

```diff
@@ -1087,7 +1087,7 @@ The best known solution for the )"
 
 std::string translate_docstring()
 {
-    return R"(__init__(udp = null_problem(), translation = [0.])
+    return R"(__init__(udp, translation)
```

Comment:
Because we do not allow default values. Those are only chosen if the default constructor is used. I guess we intend for the user to not assume any default values and assign explicitly them. The default constructor should not be called.